test/views/helpers/index.js

```diff
@@ -0,0 +1,36 @@
+var assert = require('assert');
+var helpers = require('../../../views/helpers');
+
+
+describe('views.helpers', () => {
+  describe('#contrastColor', () => {
+    it('should return "white" when the value is not present', () => {
```

Comment:
The `return` is gonna be duplicated even if the `should` is removed though, right? Still getting my head around this style of test…